app/components/search_status_component.html.erb

```diff
@@ -0,0 +1,16 @@
+<% if @developer.actively_looking? %>
+  <div class="flex items-center space-x-2">
+    <%= inline_svg_tag "icons/solid/search_circle", class: "h-5 w-5 text-green-500" %>
+    <span class="text-green-700 text-sm font-medium"><%= humanize :actively_looking %></span>
+  </div>
+<% elsif @developer.open? %>
+  <div class="flex items-center space-x-2">
+    <%= inline_svg_tag "icons/solid/search_circle", class: "h-5 w-5 text-gray-400" %>
+    <span class="text-gray-900 text-sm font-medium"><%= humanize :open %></span>
+  </div>
+<% elsif @developer.not_interested? %>
+  <div class="flex items-center space-x-2">
+    <%= inline_svg_tag "icons/solid/x_circle", class: "h-5 w-5 text-gray-400" %>
+    <span class="text-gray-900 text-sm font-medium"><%= humanize :not_interested %></span>
+  </div>
+<% end %>
```

Comment:
The html within each condition is very similar. 

You could add `icon` and `icon_attrs` methods to the `SearchStatusComponent` (similar to what was done in 
76cc310) and your erb file becomes more like:

```erb
  <div class="flex items-center space-x-2">
    <%= inline_svg_tag icon, icon_attrs %>
    <span class="text-sm font-medium text-#{@developer.actively_looking? ? "green-700" : "gray-900"} ">
      <%= humanize @developer.search_status %>
    </span>
  </div>
```

Reply:
Great call!